src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -82,14 +93,17 @@ const TeamOverview = (): JSX.Element => {
 
     return (
         <Container>
-            <Header title={`Team ${location.state.name}`} />
-            <Search
-                originalObject={allTeamMembers}
-                updateFilteredObject={setFilteredTeamMembers}
-                placeholder='Search by name'
-                searchProps={['firstName', 'lastName', 'displayName']}
-                notifyError={setSearchError}
-            />
+            <Header title='Team'  />
```

Comment:
Created a second header for the Team's name and the searchbar, for a more elegant display on the page.